### PR TITLE
FAT-1619 Cover quick marc case from Kiwi Bug Fest

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
### Purpose
The request came from the RRT channel and the Spitfire team was working to resolve this issue [MODQM-213](https://issues.folio.org/browse/MODQM-213). To cover specific case we need to create a Karate test.

### Learning
https://issues.folio.org/browse/FAT-1619

image

### CODE REVIEW

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-authority-records.feature [hunk 1]
```diff
@@ -152,19 +159,66 @@ Feature: Test quickMARC authority records
...
-    And match response.sourceRecords[1].deleted == true
+    And match response.state == "DELETED"
+    And match response.deleted == true
 
     Given path 'authority-storage/authorities', authorityIdForDelete
     And headers headersUser
     When method GET
     Then status 404
 
+  Scenario: FAT-1619 Should update record twice without any errors
```
Comment (psmagin) [inline mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-authority-records.feature:175]:
```suggestion
  Scenario: Should update record twice without any errors
```

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-authority-records.feature [hunk 2]
```diff
@@ -152,19 +159,66 @@ Feature: Test quickMARC authority records
...
+    And request record
+    When method PUT
+    Then status 202
+
+    Given path 'records-editor/records'
+    And param externalId = testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And match response.fields contains newField
```
Comment (psmagin) [inline mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-authority-records.feature:220]: Please add a check of `recordState`. It should be "ACTUAL" after both updates.

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-holdings-records.feature [hunk 3]
```diff
@@ -321,29 +368,27 @@ Feature: Test quickMARC holdings records
     Then status 422
     Then match response.message == 'Is unique tag'
 
-### Uncomment scenario when resolve bug with duplicated 852 tag
```
Comment (psmagin) [inline mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-holdings-records.feature:324]: Which bug are you referring to?

**Review by psmagin — CHANGES_REQUESTED** [review]

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-holdings-records.feature (hunk 3 above)
Comment (Shans-Kaluhin) [reply mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-holdings-records.feature:324]: Bug that allow as to create multiply 852 tags [MODQM-204](https://issues.folio.org/browse/MODQM-204).
It's already resolved, so I decided to uncomment this case

**Review by psmagin — APPROVED** [review]

**Review by YahorSviarchkou — APPROVED** [review]

**Pull request merged**